torch/_dynamo/variables/lazy.py

```diff
@@ -10,18 +10,22 @@ class LazyCache:
     """Container to cache the real VariableTracker"""
 
     def __init__(self, value, source):
-        assert source
+        if not isinstance(value, LazySymNodeString):
```

Comment:
SymNodeVariables don't have source. Not sure how to avoid the source check in a cleaner way.